develop/tutorials/articles-dxp/100-tooling/02-liferay-ide/01-installing-liferay-ide.markdown

```diff
@@ -1,39 +1,51 @@
 # Installing Liferay @ide@ [](id=installing-liferay-ide)
```

Comment:
Should the title still have the @ide@ token? In fact, why did you replace the token everywhere with "dev studio?" Wouldn't it have been better to just change the token's value from IDE to Dev Studio CE?

Reply:
@sez11a I actually did change the token. However, since both the community and enterprise versions of Eclipse are now referred to as Dev Studio now, I figured it was more natural to state "Dev Studio" most places instead of reiterating the version type over and over (Dev Studio CE). I did use the token in most articles in the beginning, but used it sparingly afterwards.

For the article titles, I removed the token for most of them. I did keep them for the main intro article and the installation articles, since I thought it was more important to state the specific version in those. That's a judgment call though.

Reply:
Got it; thanks!